sql/core/src/test/resources/sql-tests/inputs/postgreSQL/create_view.sql

```diff
@@ -41,7 +41,7 @@ DROP TABLE emp;
 -- These views are left around mainly to exercise special cases in pg_dump.
 
 -- [SPARK-19842] Informational Referential Integrity Constraints Support in Spark
-CREATE TABLE view_base_table (key int /* PRIMARY KEY */, data varchar(20));
+CREATE TABLE view_base_table (key int /* PRIMARY KEY */, data varchar(20)) STORED AS parquet;
```

Comment:
do you know how the result becomes different? This test shouldn't rely on any features of hive table.

Reply:
The output of this query is:
```
org.apache.spark.sql.AnalysisException
Hive support is required to CREATE Hive TABLE (AS SELECT);
```

Without `STORED AS parquet`, this query now uses `parquet` as table provider, so it doesn't need hive support anymore.

Maybe, we shall use `USING hive` ? But it throws something like `dataSource not found: hive`.

Reply:
Why do we test "Hive support is required to CREATE Hive TABLE" in `create_view.sql`? I believe it's a mistake, let's just update the answer file.